source/opt/debug_info_manager.h

```diff
@@ -152,11 +152,13 @@ class DebugInfoManager {
       std::unordered_set<Instruction*>* invisible_decls);
 
   // Creates a DebugValue for DebugDeclare |dbg_decl| and inserts it before
-  // |insert_before|. The new DebugValue has the same line, scope, and
-  // operands with DebugDeclare but it uses |value_id| for value. Returns
-  // the added DebugValue, or nullptr if it does not add a DebugValue.
+  // |insert_before|. The new DebugValue has the same line and scope with
+  // |scope_and_line| and it has the same operands with DebugDeclare but it
```

Comment:
s/same line and scope with |scope_and_line|/same line and scope as |scope_and_line|